app/services/feed_processor.rb

```diff
@@ -13,10 +13,9 @@ def initialize(feeds:)
   # TBD: This should receive "stale enabled" feeds; test Feed model scopes
   def perform
     feeds.each do |feed|
-      Importer.new(feed: feed).import
+      Importer.new(feed).import
       Publisher.new(posts: feed.posts.pending).publish
-    rescue StandardError
-      logger.error { "feed processing interrupted: #{feed.name} (id: #{feed.id})" }
```

Comment:
I'm temporarily removing general exceptions handler to make debugging more straightforward during the MVP implementation.

When a fallback is available, an exception should not bubble up here.